packages/zitadel-next/src/utils/session.ts

```diff
@@ -0,0 +1,22 @@
+import { createSessionServiceClient } from "@zitadel/client/v2";
+import { createServerTransport } from "@zitadel/node";
+import { Session } from "@zitadel/proto/zitadel/session/v2/session_pb";
+import { getMostRecentCookieWithLoginname } from "./cookies";
+
+const SESSION_LIFETIME_S = 3000;
+
+const transport = createServerTransport(process.env.ZITADEL_SERVICE_USER_TOKEN!, {
+  baseUrl: process.env.ZITADEL_API_URL!,
+  httpVersion: "2",
+});
+
+const sessionService = createSessionServiceClient(transport);
```

Comment:
I would careful with this pattern from inside the library; we would be back to the singleton pattern; multi-transport/tenancy wouldn't be possible

Personally, would hold off before going GA with the libraries

Reply:
you are right with this one.. how would you make this configurable? I've changed it for the function to get the whole service for now but am not so sure about that..

Reply:
For now, do not make it configurable; pass the service as an argument. Eventually, we will see the patterns, and we will know what to do about it.

Keep it simple for now. We can always add another layer of indirection wrapping the situation.